x-pack/legacy/plugins/security/public/views/oidc_implicit/oidc_implicit.tsx

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import chrome from 'ui/chrome';
+
+chrome
+  .setVisible(false)
+  .setRootTemplate('<div id="reactOIDCImplicitRoot" />')
+  .setRootController('oidc_implicit', ($scope: any, $window: any) => {
+    const urlFragment: string = window.location.hash;
+
+    // HACK: an ugly hack to overcome automatic Kibana URL fragment modifications.
```

Comment:
__note:__ that the main thing I need to get rid of (hopefully) before this PR is ready for review

Reply:
Alternatively we could just render tiny HTML page outside of Kibana context with inline "nonced" onliner script.... That way we won't have rison that garbles this right now.